### PR TITLE
Release 0.1.1

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Production Release `0.1.1`

This release branch was made off the `qa` branch.

What's going into this release?

### Updated

- Updated favicon.ico
- Skeleton load the campaign hero (DR-2787)
- Skeleton load the swim lanes (DR-2788)
- Use Repo API v2/items/featured endpoint when loading images for hero component (DR-2801)

### Added

- Added Optinmonster embed code (DR-2744)
- Added Adobe Analytics Data Layer on the home page (DR-2700)
- Added feedback button and logic for form to submit to google spreadsheet (DR-2595)
- Added newrelic agent (DR-2772)
- Added canonical link (DR-2784)
- Added caching to API routes for `/api/featuredItem ` and `/api/homepage` as well as from the client side (DR-2785)

### CODE REVIEW

**Comment by chrisb87** [comment]
We'll need to verify the task definition settings in AWS. At least `IIIF_URL` needs to be added.

**Review by croyfish — APPROVED** [review]
To the extent that I'm able to understand the changeset, it looks good.

File: log/dc.log [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{"timestamp":"2024-02-12T17:45:34.900Z","pid":83385,"level":"info","levelCode":6,"message":"Generating featured image id"}
```
[the remark this answers is not in the document]
Comment (EdwinGuzman) [reply log/dc.log]: This shouldn't be here. It was added to gitignore but I'm removing the folder now.

**Comment by EdwinGuzman** [comment]
@chrisb87 the latest task definition includes the `IIIF_URL` env variable along with others that were missing or needed to be updated.

**Review by 7emansell — APPROVED** [review]
Lgtm

**Pull request merged**